task_3

```diff
@@ -1,11 +1,10 @@
 import sys
 
 a = ""
-if sys.argv[1:] == []:
+if not sys.argv[1:]:
     print("Please use arguments next time:)\ntask_3.py *some arguments*")
     exit()
 while a != sys.argv[1:]:
```

Comment:
You have a list of various args. So, you need to check not equality but `if in`. Here you're trying to compare a list of args to the user's input. The same as in the bash task.
![image](https://user-images.githubusercontent.com/12370649/110326456-be522100-8029-11eb-9603-0312b1f79140.png)